community.txt

```diff
@@ -16,8 +16,7 @@ about SWI-Prolog or Prolog programming in general.
     (best place for issues with class room exercises that are not
     SWI-Prolog specific)
 
-    - The Prolog IRC channel ##prolog on FreeNode
-    can be accessed via [WebChat](http://webchat.freenode.net/?channels=##prolog)
+    - The Prolog IRC channel ##prolog has moved to libera.chat (server irc.libera.chat)
```

Comment:
Is there a new link?

Reply:
the old link was to webchat, giving an easy on-ramp onto IRC. libera doesn't have webchat set up yet, afaik - so the link's gone.